core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -226,6 +227,13 @@ public static GeneticProfile loadGeneticProfileFromMeta(File file) throws IOExce
         String profileDescription = properties.getProperty("profile_description");
         String geneticAlterationTypeString = properties.getProperty("genetic_alteration_type");
         String datatype = properties.getProperty("datatype");
+        String genomeBuild = properties.getProperty("reference_genome_id");
+        int referenceGenomeId = 1;
```

Comment:
good idea, will add a validation rule.

Reply:
done.

Reply:
**reference_genome_id** is already checked by the cbioportal_common.py file. 
```
    MetaFileTypes.SEG: {
        'cancer_study_identifier': True,
        'genetic_alteration_type': True,
        'datatype': True,
        'reference_genome_id': True,
        'data_filename': True,
        'description': True
    },
```
what do you think we also add **reference_genome_id** key to the meta_study file and make it an option key, since we only support one genome build per study? 
```
    MetaFileTypes.STUDY: {
        'cancer_study_identifier': True,
        'type_of_cancer': True,
        'name': True,
        'description': True,
        'short_name': True,
        'citation': False,
        'pmid': False,
        'groups': False,
        'reference_genome_id': False,
        'add_global_case_list': False
    },
```
In that way, we can also remove the following attributes
```
# species and genomic information
species=human
ncbi.build=37
ucsc.build=hg19
``` 
from the portal.properties file to support multiple genomes.

Reply:
added some comments to the new code. It will need some extra fixes.